### PR TITLE
Gh 364 env

### PR DESCRIPTION
**Pull request by m09526**

### CODE REVIEW

File: example/deployment/local-docker/example-docker-services/configure-example/Dockerfile [hunk 1]
```diff
@@ -5,4 +5,7 @@ ADD docker/bin/example-model-*-shaded.jar /tmp/
 ADD waitForHost.sh /tmp/
 ADD docker/bin/configRest.json /tmp/
 
-CMD ["/tmp/waitForHost.sh", "http://policy-service:8080/policy/v1/status", "java", "-cp", "/tmp/example-model-*-shaded.jar", "-Dpalisade.rest.config.path=/tmp/configRest.json", "uk.gov.gchq.palisade.example.config.ExampleConfigurator", "/data/employee_file0.avro"]
+ARG PALISADE_REST_CONFIG_PATH="/tmp/configRest.json"
+ENV PALISADE_REST_CONFIG_PATH=$PALISADE_REST_CONFIG_PATH
```
Comment (ac74475) [inline example/deployment/local-docker/example-docker-services/configure-example/Dockerfile]: Are both the ARG and ENV necessary?

File: example/deployment/local-docker/example-docker-services/rest-redirector/Dockerfile [hunk 2]
```diff
@@ -4,5 +4,8 @@ RUN apk add --no-cache curl
 COPY docker/bin/example-rest-redirector-*-shaded.jar /example-rest-redirector/
 ADD docker/bin/configRest.json /example-rest-redirector/
 
+ARG PALISADE_REST_CONFIG_PATH="/example-rest-redirector/configRest.json"
```
Comment (ac74475) [inline example/deployment/local-docker/example-docker-services/rest-redirector/Dockerfile]: Are both the ARG and ENV necessary?

File: example/multi-jvm-example/multi-jvm-example-docker-services/rest-redirector/Dockerfile [hunk 3]
```diff
@@ -4,5 +4,8 @@ RUN apk add --no-cache curl
 COPY docker/bin/multi-jvm-example-rest-redirector-*-shaded.jar /multi-jvm-example-rest-redirector/
 ADD docker/bin/configRest.json /multi-jvm-example-rest-redirector/
 
+ARG PALISADE_REST_CONFIG_PATH="/multi-jvm-example-rest-redirector/configRest.json"
```
Comment (ac74475) [inline example/multi-jvm-example/multi-jvm-example-docker-services/rest-redirector/Dockerfile]: Are both the ARG and ENV necessary?

File: service-impl/config-service-impl/rest-config-service/src/main/java/uk/gov/gchq/palisade/config/service/impl/ApplicationConfigV1.java [hunk 4]
```diff
@@ -20,16 +20,26 @@
...
 
     public ApplicationConfigV1() {
         super(RESOURCES);
+        //check environment variable, if that doesn't work then check system property
+        String bootStrapLocation = System.getenv(RestConfigServiceV1.BOOTSTRAP_CONFIG);
+
+        if (isNull(bootStrapLocation)) {
+            bootStrapLocation = System.getProperty(RestConfigServiceV1.BOOTSTRAP_CONFIG);
+        }
+
```
Comment (ac74475) [inline service-impl/config-service-impl/rest-config-service/src/main/java/uk/gov/gchq/palisade/config/service/impl/ApplicationConfigV1.java]: Can we add an error message for if the bootstrap location is still null after checking the env and properties

File: example/deployment/local-docker/example-docker-services/configure-example/Dockerfile (hunk 1 above)
Comment (m09526) [reply example/deployment/local-docker/example-docker-services/configure-example/Dockerfile]: will remove.

File: example/deployment/local-docker/example-docker-services/rest-redirector/Dockerfile (hunk 2 above)
Comment (m09526) [reply example/deployment/local-docker/example-docker-services/rest-redirector/Dockerfile]: will remove.

File: example/multi-jvm-example/multi-jvm-example-docker-services/rest-redirector/Dockerfile (hunk 3 above)
Comment (m09526) [reply example/multi-jvm-example/multi-jvm-example-docker-services/rest-redirector/Dockerfile]: will remove.

File: service-impl/config-service-impl/rest-config-service/src/main/java/uk/gov/gchq/palisade/config/service/impl/ApplicationConfigV1.java (hunk 4 above)
Comment (m09526) [reply service-impl/config-service-impl/rest-config-service/src/main/java/uk/gov/gchq/palisade/config/service/impl/ApplicationConfigV1.java]: Will be addressed in gh-387

**Pull request merged**